### DISCUSSION TITLE
Support for Angular 17

### DISCUSSION THREAD

**Discussion by rgustavsson** [hakimio/ngx-ui-tour#160] [General]
Hello!
Do you have any plans to support Angular 17 in the near future?

**Comment by hakimio** [comment]
Yes, it will be supported in the next major release.

**Reply by rgustavsson** [reply]
Awesome!
Do you know if there's planned release date? Based on that I will be able to judge if we should wait with our upgrade or find a workaround for now.

Cheers for the quick response!

**Reply by hakimio** [reply]
The old version should still work fine with Angular 17 (haven't tested) if you force the peer dependencies, so there isn't much reason to wait.
Apart from Angular upgrade, I also plan to add few new features. Depending on how the work on those features goes and how much spare time I have, I should release a new version sometime within the next few weeks (can't give you any precise time now).

**Reply by rgustavsson** [reply]
Your right, it still works. Did a force upgrade and it functions as expected.
Thanks for the update on next release!

**Comment by hakimio** [comment]
`v12` with Angular 17 support has been released. Check out [the release announcement](https://github.com/hakimio/ngx-ui-tour/releases/tag/v12.0.0).

**Reply by rgustavsson** [reply]
Thank you!
